tutorials/v7/draw_mt.cxx

```diff
@@ -87,9 +109,14 @@ void draw_mt()
    gEnv->SetValue("WebGui.HttpThrd","yes");
    gEnv->SetValue("WebGui.SenderThrds","yes");
 
-   std::thread thrd1(draw_canvas, "First canvas", RColor::kRed);
-   std::thread thrd2(draw_canvas, "Second canvas", RColor::kBlue);
-   std::thread thrd3(draw_canvas, "Third canvas", RColor::kGreen);
+   ROOT::EnableThreadSafety();
+
+   // workaround - dry run to load libs and create all streamers
+   // draw_canvas("Zero", RColor::kRed);
```

Comment:
Looks like it's not needed anymore? Remove?

Reply:
Ok